src/compiler/diagnosticMessages.json

```diff
@@ -1023,6 +1023,10 @@
         "category": "Error",
         "code": 2342
     },
+    "This syntax requires an imported helper named '{1}', but module '{0}' has no exported member '{1}'.": {

```

Comment:
This follows the same pattern as 2305: `"Module '{0}' has no exported member '{1}'."`